e2e/e2e-create-windows-nodepool.sh

```diff
@@ -7,24 +7,34 @@ source e2e-helper.sh
 log "Starting to create windows nodepool"
 
 if [[ "$RESOURCE_GROUP_NAME" == *"windows"*  ]]; then
-    RESOURCE_GROUP_NAME="$RESOURCE_GROUP_NAME"-"$WINDOWS_E2E_IMAGE"
+    RESOURCE_GROUP_NAME="$RESOURCE_GROUP_NAME"-"$WINDOWS_E2E_IMAGE"-v2
```

Comment:
Q: Is this used to avoid conflict with current pipelines?

Reply:
Yes. Also avoid conflict of hotfix on official branch.